src/input/mpegts/tvhdhomerun/tvhdhomerun.c

```diff
@@ -86,7 +86,7 @@ tvhdhomerun_device_class_get_title( idnode_t *in, const char *lang )
   static char buf[256];
   tvhdhomerun_device_t *hd = (tvhdhomerun_device_t *)in;
   snprintf(buf, sizeof(buf),
-           "%s - %s", hd->hd_info.friendlyname, hd->hd_info.ip_address);
+           "%s - %s", hd->hd_info.friendlyname, inet_ntoa(hd->hd_info.ip_address));
```

Comment:
Pls, use tvh's tcp_get_str_from_ip() function here... inet_ntoa might not allow recursive calls (static return buffer).